packages/core/admin/admin/src/StrapiApp.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import configureStore from './core/store/configureStore';
+import reducers from './reducers';
+import basename from './utils/basename';
+import LanguageProvider from './containers/LanguageProvider';
+// TODO remove
+import App from './containers/App';
+import Fonts from './components/Fonts';
+
+// TODO
+import translationMessages from './translations';
+
+// const App = () => 'todo';
+
+window.strapi = {
+  backendURL: process.env.STRAPI_ADMIN_BACKEND_URL,
+};
+
+class StrapiApp {
```

Comment:
ℹ️ I'm going to switch in "relou mode" over 9000: at some point we'll need automated test on this file

Reply:
Sure I'll add some!